db/citations.yml

```diff
@@ -10092,3 +10092,42 @@ zhu;gca11:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/12zhugca.pdf
   year: '2012'
+lo;nc20:
+  author:
+    - Lo, Sheng-Han
```

Comment:
all author names should be as they appear in the journal, not as they appear in our database.  This may be true here, but please check.